lua/luaconf.h

```diff
@@ -839,13 +839,24 @@ extern void longjmp(luai_jmpbuf);
 #define LUAI_TRY(L,c,a)		if (setjmp(((c)->b)) == 0) { a }
 
 /* time.h */
-#include <linux/time.h>
 #include <linux/ktime.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,6,0)
+#include <linux/time.h>
+typedef struct timespec time_t;
+#define luai_time(t)		getnstimeofday(&t)
+#else
+#ifdef LUA_32BITS
```

Comment:
#elif?

Reply:
I don't believe that will work.
Because the error generation only occurs when `LUA_32BITS `is defined, however the other definitions inside the `else` don't depend on that